src/phoenix/experimental/callbacks/llama_index_trace_callback_handler.py

```diff
@@ -53,11 +55,12 @@ def payload_to_semantic_attributes(payload: Dict[str, Any]) -> Dict[str, Any]:
     """
     attributes = {}
 
+    if EventPayload.CHUNKS in payload and EventPayload.EMBEDDINGS in payload:
+        attributes[EMBEDDINGS_TEXT] = payload[EventPayload.CHUNKS]
+        attributes[EMBEDDINGS_VECTOR] = payload[EventPayload.EMBEDDINGS]
```

Comment:
are these plural or singular?

Reply:
addressed in [2fe71e5](https://github.com/Arize-ai/phoenix/pull/1169/commits/2fe71e55efc00378f6c7b82fa6e08905b0348bfb)